flink-jepsen/src/jepsen/flink/db.clj

```diff
@@ -41,7 +41,7 @@
   {:high-availability                     "zookeeper"
    :high-availability.zookeeper.quorum    (zookeeper-quorum test)
    :high-availability.storageDir          "hdfs:///flink/ha"
-   :jobmanager.heap.size                  "2048m"
+   :jobmanager.memory.process.size        "2496m"
```

Comment:
Same here? Why is the value change "2048 -> 2496"?

Reply:
same here, it shoud be `2702m`. Though, as discussed with @GJL, it does not matter too much in this case.